Description: Support PF_RING devices
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 argus (3.0.8-1ppa1) precise; urgency=low
 .
   * Initial release  Argus 3.0.8
Author: Ponsonby Britt <bullwinkle@example.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- argus-3.0.8.orig/argus/ArgusSource.c
+++ argus-3.0.8/argus/ArgusSource.c
@@ -4382,7 +4382,9 @@ ArgusGetInterfaceStatus (struct ArgusSou
    if (device == NULL)
       return;
 
-   if ((strstr(device->name, "dag")) || (strstr(device->name, "napa"))) {
+   if ((strstr(device->name, "dag")) || (strstr(device->name, "napa"))
+|| (strstr(device->name, "dna")) || (strstr(device->name, "zc")) ||
+((strstr(device->name, "eth")) && (strstr(device->name, "@")))) {
       for (i = 0; i < src->ArgusInterfaces; i++) {
          if (src->ArgusInterface[i].ArgusPd)
             bzero ((char *)&src->ArgusInterface[i].ifr, sizeof(ifr));
